debug/v0.1/instance_openapi.yml

```diff
@@ -36,17 +36,12 @@ paths:
         '201':
           description: Successful operation
       security:
-        - provider_auth:
-          - debug
+        - provider_auth: []
 components:
   schemas:
     DebugPayload:
       type: object
   securitySchemes:
     provider_auth:
-      type: oauth2
-      flows:
-        clientCredentials:
-          tokenUrl: https://provider.example.com/oauth/token
-          scopes:
-            debug: create debug logs
+      type: http
+      scheme: bearer
```

Comment:
These wouldn't be Bearer still, would they?

Reply:
No, they would not. One of the reasons I marked this as "Draft".

There does not seem to be a standardized way to describe http signatures here, but I have seen examples of people using `scheme: signature` here, which might be better than nothing.